<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Temp Mail</class>
 <widget class="QMainWindow" name="Temp Mail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temp Mail</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 18, 29)</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWebEngineView" name="textBrowser" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>981</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
     </font>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px, solid black;
border-radius: 40px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Inbox_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 2px, solid white</string>
    </property>
    <property name="text">
     <string>ðŸ“¨Inbox messages:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshButton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Get new mail</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 0px;</string>
    </property>
    <property name="text">
     <string>ðŸ”„</string>
    </property>
   </widget>
   <widget class="QLabel" name="too_fast_label">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Too fast!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>301</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(26, 26, 36)</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>259</width>
       <height>629</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>261</width>
        <height>611</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>621</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vbox"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="copy_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	background-color: rgba(10, 0, 0, 0);
	color: white;
}
*:hover{
	color: rgb(57, 99, 182);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="empty_mail_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>921</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>              Your email is empty.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
